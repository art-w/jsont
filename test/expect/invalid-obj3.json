{ "bla": 1, "hey": "ho"
