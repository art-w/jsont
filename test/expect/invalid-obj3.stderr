jsont: [01;31mError[m: Expected [1m,[0m or [1m}[0m after [1mobject member[0m but found: [1mend of text[0m
              File "invalid-obj3.json", line 2, characters 0-1:
